tensorboard/webapp/theme/_tb_theme.template.scss

```diff
@@ -13,20 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-@import '../angular_material_theming';
-@import './tb_palette';
+@import 'tensorboard/webapp/angular_material_theming';
+
+// VARIABLE FILE IS INLINED WITH GENRULE. DO NOT IMPORT IT.
```

Comment:
1. Can you add a little more info about what "VARIABLE FILE" is to this comment (e.g., what the variables are for? palette? what the file name is, _variable.scss)? 

2. Consider make this comment more salient by adding flanking lines like `//////////////////////////////////` and exclamation points.

Reply:
Do note that the name of the file, _variable.scss, is not required. It is only useful to the genrule.